helm/prometheus-meta-operator/templates/recording-rules/service-levels.rules.yml

```diff
@@ -36,6 +36,16 @@ spec:
         class: MEDIUM
         service: kubelet
       record: raw_slo_errors
+    - expr: count(up{app="node-exporter", role!="bastion"}) by (cluster_type, cluster_id)
+      labels:
+        class: MEDIUM
+        service: node-exporter
+      record: raw_slo_requests
+    - expr: sum((up{app='node-exporter', role!="bastion"} * -1) + 1) by (cluster_type, cluster_id)
```

Comment:
Can you explain what this does? I am also confused why this is here and not under `service-level-apps`

Reply:
This counts the number of node-exporters that are down.

up == 1 if the node-exporter is up, and up == 0 if it's down. With errors, we want to have a count of all the node-exporters that are down. So, * -1 to get -1 for up node-exporters and 0 for down node-exporters, and then add 1 to get 0 for up node-exporters, and 1 for down node-exporters. Then, sum them all up to get the number of node-exporters that are down. Math!

This isn't under `service-level-apps` because that is a different SLO implementation that we're planning on removing in future.

Reply:
Okay makes sense to me - is there a way to inhibit this by now? Like this should not fire if the kubelet SLO fires.

Reply:
Not yet, but we can totally bring it in without too much pain

Reply:
Could you add what you added here as a comment to explain the SLI ?